### ISSUE TITLE
Bug in CUDA occupancy calculator wrapper

### ISSUE THREAD

**Issue by kingcrimsontianyu** [ROCm/HIP#879]
There seems to be a bug in the current master branch (190e3615), file `hip/include/hip/nvcc_detail/hip_runtime_api.h` line 1191. The last (6th) parameter `flags` should be removed, because according to CUDA runtime API documentation, this function accepts up to 5 arguments.

```
template < class T >
__host__ ​cudaError_t cudaOccupancyMaxPotentialBlockSize(int* minGridSize,
                                                        int* blockSize,
                                                        T func,
                                                        size_t dynamicSMemSize = 0,
                                                        int  blockSizeLimit = 0)
```

**Comment by gargrahul** [comment]
@kingcrimsontianyu Thanks for pointing this. I think 
https://github.com/ROCm-Developer-Tools/HIP/blob/master/include/hip/nvcc_detail/hip_runtime_api.h#L1191 should call cudaOccupancyMaxPotentialBlockSizeWithFlags. We will correct this.

**Comment by prinz7** [comment]
This Issue is fixed with #1197

**Comment by nartmada** [comment]
Closing the ticket as issues has been fixed.
